File: fiftyone/internal/features/flags.py
"""
FiftyOne feature flags.

| Copyright 2017-2026, Voxel51, Inc.
| `voxel51.com <https://voxel51.com/>`_
|
"""

from typing import Literal


FeatureFlag = Literal[
    "VFF_DYNAMIC_GROUP_ANNOTATION",
    "VFF_MULTIMODAL",
]
"""Enumeration of active feature flags."""

File: app/packages/feature-flags/src/client/flags.ts
/**
 * Enumeration of active feature flags.
 */
export enum FeatureFlag {
  VFF_MULTIMODAL = "VFF_MULTIMODAL",
  /**
   * Renders the clean-room rewrite of the dataset view bar
   * (`NewViewBar`) instead of the legacy xstate-driven `ViewBar`.
   */
  VFF_NEW_VIEW_BAR = "VFF_NEW_VIEW_BAR",
}
